src/models/filterandzoomstack.cpp

```diff
@@ -34,6 +34,8 @@
 FilterAndZoomStack::FilterAndZoomStack(QObject* parent)
     : QObject(parent)
 {
+    m_actions.disassembly = new QAction(QIcon::fromTheme(QStringLiteral("view-filter")), tr("Disassembly"), this);
```

Comment:
showing the disassembly is not a filter operation

also, as icon, what about using view-list-details instead?